dask/optimization.py

```diff
@@ -1023,3 +1023,6 @@ def __call__(self, *args):
 
     def __reduce__(self):
         return (SubgraphCallable, (self.dsk, self.outkey, self.inkeys, self.name))
+
+    def __hash__(self):
+        return hash((tuple(self.outkey, tuple(self.inkeys), self.name)))
```

Comment:
Typo, swapped parenthesis (tuple(a,b,c)) instead of tuple((a,b,c))
```suggestion
        return hash(tuple((self.outkey, tuple(self.inkeys), self.name)))
```

TypeError: tuple expected at most 1 argument, got 3